language/move-prover/tests/sources/stdlib/modules/libra_system.exp

```diff
@@ -1 +1,362 @@
+     =     at tests/sources/stdlib/modules/validator_config.move:25:5: has (entry)
+     =     at tests/sources/stdlib/modules/libra_system.move:242:45: add_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:244:32: add_validator_
+     =         validator_set_ref = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:245:64: add_validator_
+     =         discovery_set_ref = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:249:27: add_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:145:4: is_validator_ (entry)
+     =     at tests/sources/stdlib/modules/libra_system.move:146:28: is_validator_
+     =         addr = <redacted>,
+     =         validators_vec_ref = <redacted>,
+     =         size = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:147:13: is_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:151:13: is_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:153:49: is_validator_
+     =         i = <redacted>,
+     =         validator_info_ref = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:155:18: is_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:158:19: is_validator_
+     =         i = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:159:17: is_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:249:13: add_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:248:8: add_validator_
+     =     at tests/sources/stdlib/modules/libra_system.move:424:4: make_validator_info (entry)
+
+bug:  This assertion might not hold.
+
+      ┌── libra_system.bpl:1116:6 ───
+      │
+ 1116 │     assert false; // $LibraAccount_write_to_event_store not implemented
+      │      ^
+      │
+
+bug:  This assertion might not hold.
+
+      ┌── libra_system.bpl:1116:6 ───
+      │
+ 1116 │     assert false; // $LibraAccount_write_to_event_store not implemented
+      │      ^
+      │
+
+error:  A precondition for this call might not hold.
+
+     ┌── tests/sources/stdlib/modules/libra_system.move:120:5 ───
+     │
+ 120 │ ╭     public fun get_discovery_address(d: &DiscoveryInfo): &address {
+ 121 │ │         &d.addr
+ 122 │ │     }
+     │ ╰─────^
+     │
+     =     at tests/sources/stdlib/modules/libra_system.move:212:5: get_discovery_index (entry)
+     =     at tests/sources/stdlib/modules/libra_system.move:213:27: get_discovery_index
+     =         discovery_set = <redacted>,
+     =         addr = <redacted>,
+     =         len = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:214:17: get_discovery_index
+     =         i = <redacted>
+     =     at tests/sources/stdlib/modules/libra_system.move:216:54: get_discovery_index
+     =     at tests/sources/stdlib/modules/libra_system.move:120:5: get_discovery_address (entry)
```

Comment:
None of the functions in `libra_system.move` is currently specified. However, Prover produces this kind of errors. Is this expected?

Reply:
It seems that it still produces this kind of errors even with `// noverify`. I will try land this PR as it is for now.